viewer/views.py

```diff
@@ -1,35 +0,0 @@
-import json
-import os.path
-
-from django.conf import settings
-from django.http import HttpResponse, HttpResponseServerError, JsonResponse
-
-from .api.models import Metric
-
-
-def ops_lbheartbeat(request):
-    return HttpResponse('ok')
-
-
-def ops_heartbeat(request):
-    try:
-        # Check the database by counting Metrics.
-        Metric.objects.count()
```

Comment:
FTR, while there is a [check in dockerflow](https://github.com/mozilla-services/python-dockerflow/blob/edd192afdea1d4fafaedbb4672f1773e2737b783/src/dockerflow/django/checks.py#L35-L55) to test if the database connection succeeds, you can also write custom checks to check this model specifically.